gabbi/httpclient.py

```diff
@@ -29,10 +28,10 @@
 
 
 class Http(urllib3.PoolManager):
-    """A subclass of the urllib3.PoolManager to munge the data.
+    """A subclass of the ``urllib3.PoolManager`` to munge the data.
 
     This transforms the response to look more like what httplib2
-    provided when it was used as the httpclient.
+    provided when it was used as the HTTP client.
```

Comment:
Not sure whether that's what's meant here or why it's relevant at all: As someone not intimately familiar with the code's evolution, that historical note is somewhat baffling, even confusing.

Reply:
Something needs to be said there, otherwise, there's no reason for the transformation to be happening.

The transformation is happening because it used to be httplib2 that was the client and the gabbi uses those semantics internally.

So if you wanted to change this then incorporating some of that info could potentially be useful.

Reply:
In addition the use of the term 'httpclient' was to draw an association with the file that you're currently editing, not the generic notion of an HTTP client. But either is fine.

Reply:
Off the top of my head, I'm not sure how to improve this, so might as well not worry about it for the moment.